samples/react/01 SignupForm/package.json

```diff
@@ -27,7 +27,6 @@
     "@types/react-redux": "^4.4.32",
     "@types/redux": "^3.6.31",
     "@types/redux-thunk": "^2.1.31",
-    "@types/whatwg-fetch": "0.0.32",
```

Comment:
New versions of TypeScript already have `fetch` typings. It was causing duplicating export conflicts, see [this](http://stackoverflow.com/a/42429014)